core/thread/inc/ROOT/RConcurrentHashColl.hxx

```diff
@@ -20,12 +21,15 @@ class TRWSpinLock;
 
 namespace Internal {
 
-struct RHashSet;
+struct RHashMap;
+using RUidColl = std::vector<Int_t>;
```

Comment:
```suggestion
using RUidColl = std::vector<int>;
```
It's new code, no need for `_t`s.

Reply:
Maybe ... but `Int_t` (32 bits) is not the same as `int` (some number of bits)

Reply:
In new style code (classes starting with R) we don't use ROOT's _t types anymore but the standard types. So please use int or int32_t or whatever you see for.

Reply:
> So please use int or int32_t or whatever you see for.

I disagree here because this type must match what I put it (which is `Int_t`) [And `int` is definitively 'wrong' (on some platform it is `long`] [And using `int32_t` as a general rule is premature as the I/O does not support it properly (`int32_t` will be seen by the I/O as the underlying type and it will change from platform to platform and will either lead to CheckSum check failure when using ClassDef or with having to deal with (at least) 2 checksums when writing I/O rules (without a ClassDef)).

I.e. I *hope* we did not mistakenly rule that using raw `int` and `std::*_t` is okay for anything that wants to be stored in file.

Reply:
Thanks, I understand. But I wonder how much "new interface style" this class brings with it, also because it doesn't seem to be user facing? Or do you foresee use cases for users? Maybe given the fact it's Internal:: we don't actually care about any of this? :-)